src/audio/volume/volume.c

```diff
@@ -646,19 +646,21 @@ static vol_zc_func vol_get_zc_function(struct comp_dev *dev,
 static void volume_set_alignment(struct audio_stream *source,
 				 struct audio_stream *sink)
 {
-#if SOF_USE_HIFI(3, VOLUME) || SOF_USE_HIFI(4, VOLUME) || SOF_USE_HIFI(5, VOLUME)
-	/* Both source and sink buffer in HiFi 3 or HiFi4 processing version,
+	/* Both source and sink buffer in HiFi5  processing version,
+	 * xtensa intrinsics ask for 16-byte aligned.
+	 *
+	 * Both source and sink buffer in HiFi 3 or HiFi4 processing version,
 	 * xtensa intrinsics ask for 8-byte aligned. 5.1 format SSE audio
 	 * requires 16-byte aligned.
 	 */
-	const uint32_t byte_align = audio_stream_get_channels(source) == 6 ? 16 : 8;
+	const uint32_t byte_align = audio_stream_get_channels(source) == 6 ?
+		SOF_FRAME_BYTE_ALIGN_6CH : SOF_FRAME_BYTE_ALIGN;
```

Comment:
@lgirdwood I don't remember from where this align requirement for 6ch comes from. There was no discussion that I could find for it in #5266. Is it specific to peakvolume or generic for loading/storing the format in 64 bit or 128 bit chunks. If internal to peakvolume, then this SOF_FRAME_BYTE_ALIGN_6CH in common.h would make no sense.

Reply:
6ch is 5.1 via display port.